CLOUD9_SETUP.md

```diff
@@ -14,167 +14,157 @@ Note: If you encounter any issues, try checking the Troubleshooting section at t
 
 ## Setup / Test
 
-1. Create AWS Cloud9 environment in AWS Console
```

Comment:
FYI this content is the same just refactored into headings for easier permalinking